### ISSUE TITLE
Broken Configure Custom Torrent Providers

### ISSUE THREAD

**Issue by darkbroodzed** [pymedusa/Medusa#3465]
**Branch/Commit:** 1a8dc66
**OS:** Any
**What you did:** add custom rss torrent provider
**What happened:**js alert
**What you expected:** working custom rss torrent provider
**Logs:**

```
None
```
![image](https://user-images.githubusercontent.com/16042551/33989098-5b7d7a74-e0d7-11e7-8b99-61315b763c33.png)
Jackett rss url:
http://192.168.100.7:9117/api/v2.0/indexers/ettv/results/torznab/api?apikey=h6rq1vjac3idb35tkolvue760cudcfns&t=search&cat=&q=
and url works and open in browser

**Comment by medariox** [comment]
Jackett providers need to be added as `Custom Newznab Provider`.

**Comment by darkbroodzed** [comment]
> Jackett providers need to be added as Custom Newznab Provider.

And this functionality broken now: 
Jackett integration broken for providers redirecting to magnets #3435

Jackett provide rss feed so it also must be possible add it as custom torrent provider
![image](https://user-images.githubusercontent.com/16042551/33993266-e25a6d54-e0e6-11e7-9a53-8ed86527ee51.png)

So it definitely bug in medusa rss url parsing

**Comment by medariox** [comment]
Opening a new issue about something completely unrelated won't help, you need to wait for the fix.

**Comment by darkbroodzed** [comment]
>Opening a new issue about something completely unrelated won't help, you
need to wait for the fix.
???
let me rephrase:
1. i have valid rss feed with torrents files
2. i can't add it as custom torrent provider because Medusa fail to parse
valid rss

In my world its bug and its related.

**Comment by medariox** [comment]
RSS feeds from Jackett are not supported, you need to add Jackett providers as `Custom Newznab Providers`.

**Comment by darkbroodzed** [comment]
Ok. But imho it must be mentions in wiki "how custom provider work"
